.github/workflows/PR_summary.yml

```diff
@@ -67,7 +67,7 @@ jobs:
           if grep ^deprecated_module "${file}" ; then
             printf 'info: removed file %s contains a deprecation\n' "${file}"
           else
-            printf '**error**: file %s was removed without a module deprecation\n' "${file}" >> moved_without_deprecation.txt
+            printf '\n**warning**: file `%s` was removed without a module deprecation\n' "${file}" | tee -a moved_without_deprecation.txt
```

Comment:
When there are multiple removed files without deprecations, this will lead to two line breaks between entries, but only one line break between entries when there are deprecations, right? I think this might look a little strange, particularly if a PR has both.

Maybe there should just be a single extra line break / horizontal rule added if `removed_renamed_files.txt` is nonempty?